### ISSUE TITLE
[Enhancement] Add unit tests

### ISSUE THREAD

**Issue by sh7dm** [sh7dm/vue-simple-icons#5]
Add unit tests (Jest or AVA).

**Comment by gabrieledarrigo** [comment]
Ciao @sh7dm,
I added the configuration to run tests with Jest and ts-jest and they work fine.
But I have a problem: 
when I try to do a shallow mount of the component I receive this error:

```
TypeError: Object prototype may only be an Object or null: undefined
        at setPrototypeOf (<anonymous>)

       5 | import SimpleIcons from "simple-icons";
       6 |
    >  7 | @Component<VueSimpleIcon>({
         |                ^
       8 |   render(h: Function): VNode {
       9 |     const icon = SimpleIcons[this.name];
      10 |     const path = icon.svg.slice(

      at extendStatics (src/VueSimpleIcon.ts:7:16)
      at Object.<anonymous>.__extends (src/VueSimpleIcon.ts:10:9)
      at src/VueSimpleIcon.ts:25:44
      at Object.<anonymous> (src/VueSimpleIcon.ts:25:1)
      at Object.<anonymous> (test/VueSimpleIcon.test.ts:2:1)
```

I never used typescript so it's probably due to my ignorance. 
Any advice?

**Comment by sh7dm** [comment]
@gabrieledarrigo can you publish this branch?

**Comment by gabrieledarrigo** [comment]
For sure @sh7dm, here it is:

https://github.com/sh7dm/vue-simple-icons/pull/14

**Comment by sh7dm** [comment]
Closed in #14
